packages/adminrouter/extra/src/nginx.agent.conf

```diff
@@ -58,5 +58,13 @@ http {
             include common/proxy-headers.conf;
             proxy_pass http://metrics/;
         }
+
+        # Group: System
+        # Description: DC/OS version (unauthenticated)
+        location /dcos-metadata/dcos-version.json {
```

Comment:
> This would be the first common location.

There are more, i.e. `/system/(health|logs|metrics)`

> Also, is common/http.conf shared with enterprise too or just OSS?

Nope, it is not. This part is going away sooner or later.

> Looks like Enterprise copies adminrouter, so we'd need to duplicate this PR against it.

Yep.

> I don't know what state that is in.

Lua code is deduplicated, but it still waits for merging into DC/OS - I have 4 PRs immediately mergeable, and 8 that wait for former 4 as they are rebased against them. The `nginx.(master|agent).conf` files dedup will hopefully be done this/next week.
https://github.com/mesosphere/dcos-enterprise/pull/796
https://github.com/mesosphere/dcos-enterprise/pull/798
https://github.com/mesosphere/dcos-enterprise/pull/819
https://github.com/mesosphere/dcos-enterprise/pull/831
https://github.com/mesosphere/dcos-enterprise/pull/835
https://github.com/mesosphere/dcos-enterprise/pull/842
https://github.com/dcos/dcos/pull/1438
https://github.com/dcos/dcos/pull/1444
https://github.com/dcos/dcos/pull/1477
https://github.com/dcos/dcos/pull/1486
https://github.com/dcos/dcos/pull/1487
https://github.com/dcos/dcos/pull/1493

As for the change itself - could you please:
* create a Jira issue wrt. creating a test for this endpoint, and assign it to me. For now, I am writing tests but as soon as I create decent tests-writing docs I will start asking PR submitters to do it.
* create EE PR that will port the change to EE AR

Thanks!